### PR TITLE
[Inflector] Fix pluralizing words ending with "son"

### PR DESCRIPTION
**Pull request by norkunas**
| Q             | A
| ------------- | ---
| Branch?       | 4.3
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #32713
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Comment by xabbuh** [comment]
For the `3.4` branch?

**Comment by norkunas** [comment]
There is no pluralize in 3.4

File: src/Symfony/Component/Inflector/Inflector.php [hunk 1]
```diff
@@ -228,6 +228,9 @@ final class Inflector
         // bacteria (bacterium), criteria (criterion), phenomena (phenomenon)
         ['noi', 3, true, true, 'ions'],
 
+        // seasons (season)
+        ['nosaes', 6, true, true, 'seasons'],
```
Comment (pierredup) [inline src/Symfony/Component/Inflector/Inflector.php]: The issue seems to be generally related to words ending in `son`

```diff
Inflector::pluralize('treason')
- treasa
+ treasons

Inflector::pluralize('poison')
- poisa
+ poisons

Inflector::pluralize('lesson')
- lessa
+ lessons
```

Comment (norkunas) [reply src/Symfony/Component/Inflector/Inflector.php]: Thanks. Done

**Review by nicolas-grekas — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

**Review by derrabus — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @norkunas.

**Pull request merged**